src/app/pages/home/home.tsx

```diff
@@ -4,37 +4,38 @@ import { RouteUrls } from '@shared/route-urls';
 
 import { useOnboardingState } from '@app/common/hooks/auth/use-onboarding-state';
 import { useOnMount } from '@app/common/hooks/use-on-mount';
-import { useRouteHeader } from '@app/common/hooks/use-route-header';
-import { Header } from '@app/components/header';
 import { ActivityList } from '@app/features/activity-list/activity-list';
 import { AssetsList } from '@app/features/asset-list/asset-list';
 import { FeedbackButton } from '@app/features/feedback-button/feedback-button';
 import { InAppMessages } from '@app/features/hiro-messages/in-app-messages';
 import { homePageModalRoutes } from '@app/routes/app-routes';
 import { ModalBackgroundWrapper } from '@app/routes/components/modal-background-wrapper';
 
+// import { Header } from '@app/ui/components/containers/headers/header';
+import { HomeLayout } from '../../ui/components/containers/home.layout';
 import { CurrentAccount } from './components/account-area';
 import { HomeTabs } from './components/home-tabs';
-import { HomeLayout } from './components/home.layout';
 
 export function Home() {
   const { decodedAuthRequest } = useOnboardingState();
 
   const navigate = useNavigate();
 
-  useRouteHeader(
-    <>
-      <InAppMessages />
-      <Header />
-    </>
-  );
+  // useRouteHeader(
```

Comment:
@kyranjamie do you know how to test `<InAppMessages />` or if we still need it? 

I think just loading it in this component rather than store it in the header state shouldn't change much but I wonder if it can be deprecated. 

Looking at [this PR](https://github.com/leather-wallet/extension/pull/3219) where we added it and it seems a bit outdated now

Reply:
Yes, let's not remove this feature. These allow to use share updates to the wallet without a release.

Reply:
To test, you can add message to the JSON config

Reply:
OK great, thanks!

Reply:
I tested this by hard-coding a message in `InAppMessages`:
```
export function InAppMessages(props: FlexProps) {

  // const messages = useRemoteLeatherMessages();
  const messages = [
    {
      id: '1',
      chainTarget: 'all',
      purpose: 'info',
      title: 'API experiencing slowness',
      text: 'Balances and transactions may take longer to update',
      learnMoreUrl: 'https://stacks.co',
      publishedAt: '2021-09-27T07:57:37.397Z',
      dismissible: false,
    },
  ];
```

I tried just adding one into the messages in `wallet-config.json` but it wasn't returned by `useRemoteLeatherMessages` on this branch or on `dev`. 

Here is how both are looking. Maybe we should add a better design for these  

**DEV**
![Screenshot 2024-02-27 at 06 27 05](https://github.com/leather-wallet/extension/assets/2938440/b00af12e-bb9f-4b77-acf5-daf0762dc044)


**Containers branch**
![Screenshot 2024-02-27 at 06 28 19](https://github.com/leather-wallet/extension/assets/2938440/f45a16db-c288-4bad-924d-728c1a572f36)

CC @fabric-8 @mica000